dbt/models/curated/models.yml

```diff
@@ -70,7 +70,7 @@ models:
             - url
             - club_id
       - dbt_expectations.expect_table_row_count_to_be_between:
-          min_value: 400
```

Comment:
Your changes are not affecting the `clubs` model, hence a change in the tests should not be required right?

Reply:
just mistake, sorry)